vault-sync-secrets/Dockerfile

```diff
@@ -0,0 +1,31 @@
+# Use the official Ubuntu base image
+FROM ubuntu:latest
```

Comment:
Try a lighter image?  debian-slim, alpine, rhel-ubi?

Reply:
I will remove this, this should not have been committed, this is all about crunchy , I am sorry